### PR TITLE
[Communication] - Auth - Using BearerTokenCredentialPolicy instead of CommunicationAuthenticationPolicy

### PR DESCRIPTION
**Pull request by JoshuaLai**
POC to remove the policy that is duplicated by BearerTokenCredentialPolicy

### CODE REVIEW

File: eng/common/pipelines/templates/steps/prepare-pipelines.yml [hunk 1]
```diff
@@ -132,6 +132,6 @@ steps:
         --no-schedule
         ${{parameters.TestsConventionOptions}}
       displayName: Create Live Test pipelines for private repository
-      condition: ne(variables['TestsConventionOptions'],'')
+      condition: ne(${{parameters.TestsConventionOptions}},'')
```
Comment (sacheu) [inline eng/common/pipelines/templates/steps/prepare-pipelines.yml:135]: why we change from variables to parameters?

File: eng/common/scripts/Update-ChangeLog.ps1 [hunk 2]
```diff
@@ -12,41 +12,46 @@ param (
   [String]$ServiceDirectory,
   [Parameter(Mandatory = $true)]
   [String]$PackageName,
-  [boolean]$Unreleased=$True,
-  [boolean]$ReplaceLatestEntry = $False,
+  [String]$Unreleased=$True,
```
Comment (sacheu) [inline eng/common/scripts/Update-ChangeLog.ps1:15]: why we need to change the type of these variable?

**Comment by sacheu** [comment]
why this PR includes empty files like sdk/communication/AzureCommunicationChat/Tests/Util/Recordings/sendTypingNotification.json

**Comment by JoshuaLai** [comment]
Looks like there was an issue with my rebase. I will open up another PR for this

**Pull request closed without merging**